src/robusta/cli/main.py

```diff
@@ -166,9 +169,8 @@ def gen_config(
                 )
             )
         )
-        if not verify_slack_channel(slack_api_key, cluster_name, slack_channel):
-            typer.secho(f"\nInstallation Aborted.", fg=typer.colors.RED)
-            return
+        while not verify_slack_channel(slack_api_key, cluster_name, slack_channel, slack_workspace):
```

Comment:
I think this is a bug. You've already written the slack channel to sinks_config above using the invalid value.

Reply:
Good catch. Fixing